libs/libwebsockets/Makefile

```diff
@@ -46,29 +36,59 @@ define Package/$(PKG_NAME)/Default
 	MAINTAINER:=Karl Palsson <karlp@remake.is>
 endef
 
-define Package/libwebsockets-openssl
+define Package/$(PKG_NAME)-nossl
+	$(call Package/$(PKG_NAME)/Default)
+	TITLE += (no SSL/TLS)
+	VARIANT:=nossl
+endef
+
+define Package/$(PKG_NAME)-openssl
 	$(call Package/$(PKG_NAME)/Default)
 	TITLE += (OpenSSL)
 	DEPENDS += +libopenssl
 	VARIANT:=openssl
 endef
 
-define Package/libwebsockets-cyassl
+define Package/$(PKG_NAME)-cyassl
 	$(call Package/$(PKG_NAME)/Default)
 	TITLE += (CyaSSL)
 	DEPENDS += +libcyassl
 	VARIANT:=cyassl
 endef
 
+define Package/$(PKG_NAME)-full
+	$(call Package/$(PKG_NAME)/Default)
+	TITLE += (OpenSSL + extras)
+	DEPENDS += +libopenssl \
```

Comment:
You can use DEPENDS +=, you don't need to hack around with \ and is it even too long for a single line anyway?